cinema_service/settings.py

```diff
@@ -137,3 +137,9 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+REST_FRAMEWORK = {
+    "DEFAULT_AUTHENTICATION_CLASSES": [
+        "rest_framework.authentication.TokenAuthentication",
+    ]
+}
```

Comment:
You can also add here DEFAULT_PERMISSION_CLASSES and set it to the most common permission. For viewset that don't need it, you can override permission_classes